client-v2/src/services/faciaApi.ts

```diff
@@ -68,6 +68,7 @@ function fetchFrontsConfig(): Promise<FrontsConfig> {
         const front: FrontConfig = {
           ...json.fronts[id],
           id,
+          displayName: id,
```

Comment:
Why do we need a display name here?

Reply:
Ah good point that was from a PR where I considered adding it as a normalisation step but then thought that might be a bit confusing. Will remove.

Reply:
Oops, that wasn't even in this PR 😨